plugins/package-managers/bazel/src/main/kotlin/Bazel.kt

```diff
@@ -139,7 +173,29 @@ class Bazel(override val descriptor: PluginDescriptor = BazelFactory.descriptor)
         val moduleMetadata = Parser(definitionFile.readText()).parse()
         val depDirectives = moduleMetadata.dependencies.associateBy { "${it.name}@${it.version}" }
         val archiveOverrides = getArchiveOverrides(projectDir)
-        val scopes = getDependencyGraph(projectDir, depDirectives, archiveOverrides)
+        var (mainModule, scopes) = getDependencyGraph(
+            projectDir,
+            depDirectives,
+            archiveOverrides
+        )
+
+        scopes = scopes.toMutableSet()
```

Comment:
IMO it's a bit unclean to make both `mainModule` and `scope` non-final `var`iables in the destructing declaration if only `scope` needs to be mutable (but could still be a final `val`ue instead of a variable).

Why not simply make `getDependencyGraph()` return a `MutableSet`?